### PR TITLE
#101 Corrections on Trace following 478 in Dynawo.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -94,6 +94,8 @@ list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share/cmake")
 list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/cmake")
 list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/share/cmake")
 
+set(BOOST_ROOT ${DYNAWO_HOME})
```
Comment (lecourtoisflo) [inline CMakeLists.txt:97]: To comply with "standard" way to use "var_ROOT" variables, dynawo_home is only the default value o fBoost_root and log a warning or error if value doesn't match dynawo home

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

**Comment by lecourtoisflo** [comment]
Also, Boost ROOT should be removed from the README

File: CMakeLists.txt (hunk 1 above)
Comment (gautierbureau) [reply CMakeLists.txt:97]: done

**Comment by gautierbureau** [comment]
> Also, Boost ROOT should be removed from the README

done

**Review by lecourtoisflo — APPROVED** [review]

**Pull request closed without merging**